### PR TITLE
COMPONENT_BlueNRG_MS creation in mbed-os

### PR DESCRIPTION
**Pull request by jeromecoutant**
Hi

BlueNRG chip is now a mbed-os BLE component.

PR needed for:
https://github.com/ARMmbed/mbed-os/pull/12456

### CODE REVIEW

**Comment by jeromecoutant** [comment]
@LMESTM 
@0xc0170 
@pan-

**Comment by jeromecoutant** [comment]
@0xc0170 could you start review for this one in order to restart ARMmbed/mbed-os#12456 ?
Thx

**Comment by MarceloSalazar** [comment]
@0xc0170 please review

**Comment by 0xc0170** [comment]
This needs a rebase, but still @ARMmbed/mbed-os-pan needs to review, will poke the team

File: BLE_Button/mbed_app.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
     "config": {
         "ble_button_pin_name": {
             "help": "The pin name used as button in this application",
+            "value": "BUTTON1"
```
Comment (LDong-Arm) [inline BLE_Button/mbed_app.json]:
```suggestion
            "value": "BUTTON1",
```

File: BLE_GAPButton/mbed_app.json [hunk 2]
```diff
@@ -2,6 +2,7 @@
     "config": {
         "ble_button_pin_name": {
             "help": "The pin name used as button in this application",
+            "value": "BUTTON1"
```
Comment (LDong-Arm) [inline BLE_GAPButton/mbed_app.json]:
```suggestion
            "value": "BUTTON1",
```

**Comment by jeromecoutant** [comment]
Rebase done

**Comment by jeromecoutant** [comment]
@LDong-Arm Could you review ?
Thx

**Comment by LDong-Arm** [comment]
> @LDong-Arm Can this proceed to be integrated?

@0xc0170 This depends on https://github.com/ARMmbed/mbed-os/pull/12456 being merged first, otherwise the examples here won't compile

**Comment by jeromecoutant** [comment]
chicken and egg....

**Comment by 0xc0170** [comment]
> @0xc0170 This depends on ARMmbed/mbed-os#12456 being merged first, otherwise the examples here won't compile

Rather break this example for a minute than Mbed OS master (its very  sensitive these days to CI failures as there were quite a few). I would suggest to merge this, PR to Mbed OS right away (tested) and retest the example. Including manual example test that all is fine for this example with the PR from Mbed OS.

**Comment by 0xc0170** [comment]
@ARMmbed/mbed-os-pan Who can approve and merge this one?

**Comment by MarceloSalazar** [comment]
@ARMmbed/mbed-os-pan can you please review?

**Comment by pan-** [comment]
@jeromecoutant Could you rebase, I will merge then.

**Comment by jeromecoutant** [comment]
> @jeromecoutant Could you rebase, I will merge then.

Done

**Comment by 0xc0170** [comment]
@pan- Can we merge this?

**Pull request merged**